ui/transaction.go

```diff
@@ -0,0 +1,225 @@
+package ui
+
+import (
+	"fmt"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+	"github.com/decred/dcrd/dcrutil/v2"
+	"github.com/raedahgroup/dcrlibwallet"
+	"github.com/raedahgroup/godcr/ui/decredmaterial"
+	"github.com/raedahgroup/godcr/wallet"
+)
+
+const (
+	pageInset        = 220
+	pageContentInset = 15
+	rowGroupInset    = 20
+)
+
+var (
+	transactionPageContainer    = &layout.List{Axis: layout.Vertical}
+	transactionInputsContainer  = &layout.List{Axis: layout.Vertical}
+	transactionOutputsContainer = &layout.List{Axis: layout.Vertical}
+)
+
+func (win *Window) TransactionPage() {
```

Comment:
This isn't a page but a dialog, appropriate names should be used to avoid confusion. So this would be TransactionDetailDialog. The file name should be renamed as well, "transaction_detail_dialog" would work better than just "transaction".

Reply:
Okay